content/blog/multicloud-app/index.md

```diff
@@ -0,0 +1,118 @@
+
+We will use the cloud SDKs to provision the managed Kubernetes clusters. Given
+that we're working with real code, we are afforded developer benefits such as: 
+code linting, type checking, IDE hints and completion,
+abstractions and inheritance.
+
+Leveraging these development features creates the opportunity to encapsulate
+the finer-grained details and settings, and expose the capability to create
+clusters as simple as the following code:
+
+![](clusters.png)
+
+## Workload Deployment
+
+Once the clusters are provisioned, we can leverage the
+[`pulumi/kubernetes`][pulumi-kubernetes] SDK to manage the Kubernetes
+**workloads** that will be deployed into the clusters.
+
+The `pulumi/kubernetes` SDK uses the official Kubernetes [client-go][client-go]
+library to interact with Kubernetes. Therefore, Pulumi can work pretty
+much anywhere `kubectl` works, even if Pulumi was not used to create the cluster.
+
+![](forloop.png)
+
+## Summary
+
+As shown in the code samples, it becomes relatively easy to provision and
+manage Kubernetes clusters, as well as deploy workloads to the cluster
+regardless if they are managed by a cloud provider, or self-managed.
+
+The various SDKS allow you to leverage industry standard best-practices and 
+defaults, in addition to allowing you to further configure and customize how your clusters
+and apps are managed.
+
+Testing apps across various providers in this form allows you to abstract away
+the details of provider specific implementations, and focus on how your app
+operates in the various contexts.
+
+## Learn More
+
+If you'd like to learn about Pulumi and how to manage your
+infrastructure and Kubernetes through code, [get started today]({{< relref "/docs/quickstart" >}}). Pulumi is open source and free to
+use.
+
+For further examples on how to use Pulumi to create Kubernetes
+clusters, or deploy workloads to a cluster, check out the rest of the
+[Kubernetes tutorials]({{< relref "/docs/reference/tutorials/kubernetes" >}}).
+
+As always, you can check out our code on
+[GitHub](https://github.com/pulumi), follow us on
+[Twitter](https://twitter.com/pulumicorp), subscribe to our [YouTube
+channel](https://www.youtube.com/channel/UC2Dhyn4Ev52YSbcpfnfP0Mw), or
+join our [Community Slack](https://slack.pulumi.com/) channel if you have
+any questions, need support, or just want to say hello.
+
+If you'd like to chat with our team, or get hands-on assistance with
+migrating your existing configuration code to Pulumi, please don't hesitate to [drop us a line]({{< relref "/contact" >}}).
+
+We also encourage you to watch Pulumi team member [Levi Blackstone][levi-blackstone]
+demo this post in an episode of the [Kubernetes Community Meeting](https://kubernetes.io/community).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EyW2m5Xa_BQ?start=67" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
We should use [the YouTube shortcode](https://github.com/pulumi/docs/blob/master/BLOGGING.md#video) for these:

```suggestion
{{< youtube "EyW2m5Xa_BQ?rel=0&start=67" >}}
```

(Ensures all the sizing happens correctly.)